docs/design-language.md

```diff
@@ -330,3 +330,70 @@ There are four types of notifications available to use:
+
+### Type Scale
+
+<div class="typography buttons-box">
+    <h1>h1. Heading (2rem/32px, weight: 300)</h1>
+    <h2>h2. Heading (1.75rem/28px, weight: 300)</h2>
+    <h3>h3. Heading (1.625rem/26px, weight: 400)</h3>
+    <h4>h4. Heading (1.375rem/22px, weight: 400)</h4>
+    <h5>h5. Heading (1.250rem/20px, weight: 600)</h5>
+    <h6>h6. Heading (1rem/16px, weight: 600)</h6>
+    <h6 class="subtitle1">subtitle1 (1rem/16px, weight: 400, letter-spacing: 0.00938rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur</h6>
+    <h6 class="subtitle2">subtitle2 (0.875rem/14px, weight: 600, letter-spacing: 0.01071rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur</h6>
+    <p class="body1">body1 (0.875rem/14px). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.</p>
+    <p class="body2">body2 (0.75rem/12px). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.</p>
+    <p class="caption">Caption text (0.875rem/14px)</p>
+</div><br />
+
+### Font Stack
+
+Our current font stack is <em>"Segoe UI, Helvetica Neue, Roboto, Helvetica, Arial, sans-serif"</em>. **<a href="https://learn.microsoft.com/en-us/typography/font-list/segoe-ui" target="_blank">Segoe UI</a>** is installed by default on Windows. MacOS/iOS will take advantage of Helvitca Neue and Roboto is Android's default font. **Font sizes rely on the browser default font-size of 16px. This value should not be changed.**
+
+### Headings
+
+Headings are the largest text on the screen and are used for titles. They help organize content and demonstrate importance using a hierarchy that ranges from 1-6. For more information regarding headings please see **<a href="https://www.w3.org/WAI/tutorials/page-structure/headings/" target="_blank">W3C</a>** and **<a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements" target="_blank">MDN Web Docs</a>**.
+
+#### Considerations when using Headings
+
+-   Do not alter the weight of headings using CSS or HTML tags.
+-   Avoid skipping heading levels. Maintain an appropriate logical hierarchy for each section of an app.
+-   In most cases, such as in Studio Web, the first heading in a Panel component should be an H2 (H1 is reserved for the app title).
```

Comment:
Do we still need this if we are referring people to the W3C and MDN sites for best practices? Plus not all layouts will have panels.

Reply:
Updated. Added a point about multiple H1 tags. I feel it's important enough to add there as not everyone will click on the external link to read more.

Reply:
I like it.